### PR TITLE
Removes the shuttle timid var

### PR DESCRIPTION
**Pull request by ninjanomnom**
Now that shuttles are all loaded via template we no longer have a need for the timid var on shuttles.

Well not all shuttles, it seems I forgot to template the backup shuttle so it wouldn't have been working until now. This got fixed here as well.

### CODE REVIEW

File: _maps/map_files/generic/CentCom.dmm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```
Comment (SpaceManiac) [inline _maps/map_files/generic/CentCom.dmm]: rogue fastdmm detected

File: _maps/shuttles/pirate_default.dmm [hunk 2]
```diff
@@ -660,7 +660,6 @@
 	name = "Pirate Ship";
 	port_direction = 8;
 	preferred_direction = 1;
-	timid = 0;
```
Comment (SpaceManiac) [inline _maps/shuttles/pirate_default.dmm]: Have you made sure this still works?

Comment (ninjanomnom) [reply _maps/shuttles/pirate_default.dmm]: I tested a lot of shuttles so I think I did this one but I cant remember atm. I'll check when I get home.

**Comment by ninjanomnom** [comment]
Tested and fixed the minor issue with it.

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**